### ISSUE TITLE
subseq on displaced arrays doesn't work

### ISSUE THREAD

**Issue by dkochmanski** [drmeister/clasp#286]
test case:

``` common-lisp
(defparameter *string*
      (let ((arr (make-array 5
                             :element-type
                             #+clasp 'base-char
                             #-clasp 'character
                             :initial-contents "abcde")))
        (make-array 3
                    :element-type
                    #+clasp 'base-char
                    #-clasp 'character
                    :displaced-to arr
                    :displaced-index-offset 1)))

*STRING*
> (copy-seq *string*)

Condition of type: SIMPLE-PROGRAM-ERROR
File(../../include/clasp/core/lispVector.h) lineNumber(105): Subclass[VECTOR-DISPLACED] must implement method[subseq]

Available restarts:
(use :r1 to invoke restart 1)

1. (RESTART-TOPLEVEL) Go back to Top-Level REPL.

Broken at frame[105] CALL-WITH-VARIABLE-BOUND.
 File: #P"/data/Warsztat/Repozytoria/clasp/src/core/compiler.cc" (Position #37160)
```

**Comment by Shinmera** [comment]
Confirmed on dcc6f10e04e998f628a90922797eeca6dd8b68f9

**Comment by Bike** [comment]
Seems to be fixed in b83afab.
